appinventor/components-ios/src/FileScope.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  FileScope.swift
+//  AIComponentKit
+//
+//  Created by Amelia Xiang Zhang on 8/9/23.
+//  Copyright © 2023 Massachusetts Institute of Technology. All rights reserved.
+//
+
+import Foundation
+
+public class FileScope: OptionList {
```

Comment:
We should double check, but I think this needs to be annotated `@objc` so that it can be visible to the Scheme side.